test/setup.js

```diff
@@ -0,0 +1,2 @@
+import {initialize} from 'aurelia-pal-browser';
+initialize();
```

Comment:
I'm not opposed to using pal-browser in tests, but it doesn't seem necessary here. For unit tests I think it'd be better to mock the necessary components instead of using the pal. Also, the current tests don't require it anyway since we're testing the `_createOrigin` private helper, which does not read `location`.